app/view/panel/layer/MetadataController.js

```diff
@@ -58,7 +58,12 @@ Ext.define('MoMo.admin.view.panel.layer.MetadataController', {
                             maxX: maxX,
                             maxY: maxY
                         });
-//                        view.setLoading(false);
+                        // coordinates retrieved here are always in EPSG:3857
+                        var combo = Ext.ComponentQuery.query(
+                            'combo[name=metadata-geography-projection]')[0];
```

Comment:
Could you possibly query relatibe to your own panel (layermetadata) instead of globally?